### DISCUSSION TITLE
Can an HDD be used to run a Bastyon Node?

### DISCUSSION THREAD

**Discussion by spflanze** [pocketnetteam/pocketnet.core#695] [Q&A]
I am interested in setting up a Bastyon node. According to:
https://bastyon.com/docs/en/users/0.1/node/install.html
I will need 500GB SSD. My computer system is setup with 2TB NVME, and 10TB HDD. In the NVME is my Ubuntu 22.04 OS, and all the application executables, and the boot partition, and everything in the /root except /home (all of NVME is in the ZFS root pool). The /home directory (the ZFS home pool) points to my entire HDD.

500GB is more than I am willing to dedicate from my NVME. I have plenty of HDD space I would can dedicate 500GB from for this node. But the Bastyon instructions say I need SSD. I intend to acquire an energy efficient computer to dedicate to running this node. Meanwhile I would like to get started by using my HDD I have. What difficulties would an HDD, instead of an SSD, cause?

The computer I have consumes a lot of power (160W), so would also appreciate any recommendations on energy efficient Linux compatible computers for node use.

**Comment by andyoknen** [comment]
Hi! In general, there is no strict requirement to use SSD, not HDD - this recommendation is due to the fact that people often use simple and cheap HDDs for storing data that do not provide high speed. Since the Bastion database is organized on sqlite, and all database operations are performed directly on disk (with a relatively small cache in RAM).

If you have a good HDD and a fast connection to it, try it! One person, Network_Support (nickname from Bastion), successfully launches a node on his fast HDD and is very happy;)

**Reply by spflanze** [reply]
It is a broadband Internet connection with a Ethernet cable to the router.

The computer the node is to run on has an encrypted ZFS file system, the node data is to be on two 12 TB HDD that are software raid 1 mirrored together for fault tolerance. Both of these HDD are:
Model Family:     HGST Ultrastar DC HC520 (He12)
Device Model:     HGST HUH721212ALE600

**Reply by andyoknen** [reply]
This is a good configuration! How does the node behave?
